organization_integration_test.go

```diff
@@ -263,6 +281,20 @@ func TestOrganizationsUpdate(t *testing.T) {
 		assert.EqualError(t, err, ErrInvalidOrg.Error())
 	})
 
+	t.Run("with default execution mode and without agent pool", func(t *testing.T) {
```

Comment:
test case summary could be clearer, it seems to be doing something in the line of
```suggestion
	t.Run("when specifying remote execution mode with default agent pool", func(t *testing.T) {
```

Reply:
Good catch!